src/components/security_manager/test/ssl_context_test.cc

```diff
@@ -294,10 +310,10 @@ class SSLTestParam : public testing::TestWithParam<ProtocolAndCipher> {
       mock_crypto_manager_settings_;
   utils::SharedPtr<NiceMock<security_manager_test::MockCryptoManagerSettings> >
       mock_client_manager_settings_;
-  security_manager::CryptoManager* crypto_manager;
-  security_manager::CryptoManager* client_manager;
-  security_manager::SSLContext* server_ctx;
-  security_manager::SSLContext* client_ctx;
+  security_manager::CryptoManager* crypto_manager = NULL;
```

Comment:
@pestOO It is C++11, we should not initialize members at their declaration in class